torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2015,6 +2015,27 @@ def sample_inputs_logsumexp(self, device, dtype, requires_grad):
 
     return tuple(samples)
 
+def sample_inputs_like_fns(self, device, dtype, requires_grad, full_like=False):
```

Comment:
All sample inputs functions should take `**kwargs`

```suggestion
def sample_inputs_like_fns(self, device, dtype, requires_grad, full_like=False, **kwargs):
```

Reply:
Some sample inputs take optional kwargs that tests can opportunistically set:

https://github.com/pytorch/pytorch/blob/6d4d636d66774eb803cffedd267849a8b8a3b536/torch/testing/_internal/common_methods_invocations.py#L4353